packages/flex-plugin-webpack/src/webpack/webpack.dev.ts

```diff
@@ -48,17 +62,18 @@ export const _getStaticConfiguration = (config: Configuration): Configuration =>
 // eslint-disable-next-line import/no-unused-modules
 export const _getJavaScriptConfiguration = (config: Configuration): Configuration => {
   const socket = env.getWSSocket();
-  config.injectClient = false;
-  config.serveIndex = false;
 
+  (config.static as Static).serveIndex = true;
   // We're using native sockjs-node
-  config.transportMode = 'ws';
-  config.sockHost = socket.host;
-  config.sockPath = socket.path;
-  config.sockPort = socket.port;
+  config.webSocketServer = 'ws';
+  (config.client as ClientConfiguration).webSocketURL = {
+    hostname: socket.host,
+    pathname: socket.path,
+    port: socket.port,
+  };
 
   // Hot reload
-  config.hot = true;
+  config.hot = false;
```

Comment:
Does this mean hot reload wont work in dev server?

Reply:
I guess hot reload doesn't work with current setup as well